variables.tf

```diff
@@ -421,12 +421,12 @@ variable "dcos_cluster_docker_credentials_enabled" {
 }
 
 variable "dcos_master_list" {
-  default     = ""
+  default     = []
```

Comment:
I see this is currently requiring a list: https://github.com/dcos-terraform/terraform-template-dcos-core/pull/6/files#r267001296